### DISCUSSION TITLE
Build error

### DISCUSSION THREAD

**Discussion by remlostime** [pointfreeco/swift-composable-architecture#937] [Q&A]
**Describe the bug**
Build error

**To Reproduce**
Open the SwiftUICaseStudies.xcodeproj

**Expected behavior**
build successfully

**Screenshots**
![image](https://user-images.githubusercontent.com/3702498/147127552-74dd782c-fe8a-4346-a9bb-00fa76a5a257.png)

**Environment**
  - swift-composable-architecture version 0.31
  - Xcode 13.0
  - Swift 5
  - OS: [e.g. iOS 14]

**Additional context**
Add any more context about the problem here.

**Comment by stephencelis** [comment]
Hi @remlostime! While the library itself still supports Xcode 12.4 and iOS 13, we try to keep the demo apps and case studies up-to-date with the latest stable versions of Xcode and iOS. If you'd like to give them a try, please use Xcode 13.2 and iOS 15 😄

**Comment by mbrandonw** [comment]
@remlostime I'm going to convert this to a discussion since it isn't exactly an issue with the library. If you have further comments/suggestions please do post in the discussion.
